src/App.tsx

```diff
@@ -1,12 +1,63 @@
 import React from 'react';
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
+import { GoodsList } from './GoodsList';
+import { getAll } from './api/goods';
 // or
 // import * as goodsAPI from './api/goods';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goods: [],
+  };
+
+  getGoods = async () => {
+    try {
+      const allGoods = await getAll();
+
+      this.setState({ goods: allGoods });
+    } catch (error) {
+      throw new Error(`Error - ${error}`);
```

Comment:
And the execution of application stops, cause of unhandled error, just show some info for user in your render method if something went wrong